new_docs/DOCSTRUCTURE.md

```diff
@@ -0,0 +1,53 @@
+# doc-structure
+this is a repository for the new documentation structure
+
+
+-----------------------------------------------------------------------------
+
+
+Chapter 1 Concepts of thin-edge.io 
+
+(In this chapter terminology and concepts of thin-edge.io should be explained. Not only text should be used but also (simple/schematic) images. See the cumulocity concepts guide for examples. What we have now is too developer oriented, concepts should be more general described
```

Comment:
Should this stay in the document? If so, could it be rephrased to be more guiding and less accusing?

e.g. "Document writers should try to strive for easy to understand prose accompanied with images that illustrate their points. At the forefront should be readability of targeted users and not the developers of thin-edge.io themselves. For a guide to component development see [here]"